src/server.c

```diff
@@ -252,6 +262,8 @@ static void thread_process_listeners_LH(qd_server_t *qd_server)
         ctx->user_context = 0;
         ctx->link_context = 0;
         ctx->ufd          = 0;
+        ctx->uid          = qd_server->next_connection_id; // Increment the connection id so the next connection can use it
+        qd_server->next_connection_id++;
```

Comment:
You can combine these into one statement:  ctx->uid = qd_server->next_connection_id++;